cms/admin/permissionadmin.py

```diff
@@ -31,6 +31,29 @@ class TabularInline(admin.TabularInline):
     pass
 
 
+def users_exceed_threshold():
+    """
+    Check if the number of users exceed the configured threshold. Only bother
+    counting the users when using an integer threshold, otherwise return the
+    value of the setting to avoid a potentially expensive DB query.
+    """
+    threshold = get_cms_setting('RAW_ID_USERS')
+
+    # Don't bother counting the users when not using an integer threshold
+    if threshold is True or threshold is False:
```

Comment:
I feel like there should be some more explicit type checking done here if the type we're interested in is an `int`